.github/workflows/e2e.yml

```diff
@@ -1,22 +1,26 @@
 ---
+name: End to End Tests
+
 on:
   pull_request:
+    branches:
+      - '**'
```

Comment:
Why is this needed?

Reply:
I was just trying to make it consistent, but you're right it's likely better to make it consistent by removing vs adding.